store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9381,11 +9381,7 @@ TODO: delete them permanently from here
   <desc>Whether to enable zimbra network new generation mobile sync module.</desc>
 </attr>
 
-<attr id="2119" name="zimbraJWTKey" type="ostring" max="128" immutable="1" cardinality="single" optionalIn="account"  flags="domainAdminModifiable" since="8.8.0">
-  <desc>JWT secret key</desc>
-</attr>
-
-<attr id="2120" name="zimbraJWTLifetime" type="duration"  max="3600" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" callback="PositiveTimeInterval" since="8.8.0">
+<attr id="2119" name="zimbraJWTLifetime" type="duration"  max="3600" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" callback="PositiveTimeInterval" since="8.8.0">
```

Comment:
auth token and jwt can have different lifetime, so can't use auth token lifetime.